src/styles/blocks/container.scss

```diff
@@ -0,0 +1,17 @@
+.container {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
+  justify-content: center;
+
+  &__box {
+    height: 300px;
+    text-align: center;
+    line-height: 300px;
```

Comment:
Here `line-height` already sets the height of the box to 300px, so the `height `property is redundant. To check that open your demo link in chrome developer's tools and uncheck the property to see if anything changes.